### ISSUE TITLE
manual check interactive elements indicate their purpose and state

### ISSUE THREAD

**Issue by a20behta** [HGustavs/LenaSYS#11904]
related to issue #11794 

check that all elemets have their purpose indicated and that their state is visible, mainly used for screen readers etc

**Comment by a20caros** [comment]
<img width="265" alt="image" src="https://user-images.githubusercontent.com/81676918/166417739-dee67a8c-f034-4d27-95e9-768c05bbfebd.png">
When tab is used to check if an element indicates a purpose, the symbol does not indicate that it is interactable

**Comment by a20caros** [comment]
<img width="695" alt="image" src="https://user-images.githubusercontent.com/81676918/166425872-ad9fc71c-0a95-4fe0-ae33-147abaa9b787.png">
The login button cannot be interacted with when using the keyboard to navigate and thus the screen reader does not find the element. It is also not possible to navigate when you are logged in to be able to navigate to the log out button. You can only navigate to the name of the person who is logged in. Which can be seen in the image below where the name of the logged in has a white frame around it that says that the user interacts with it.
<img width="456" alt="image" src="https://user-images.githubusercontent.com/81676918/166427011-08a6d532-c9a7-4dce-8615-479b2119cce9.png">

**Comment by a20caros** [comment]
<img width="828" alt="image" src="https://user-images.githubusercontent.com/81676918/166430551-144d6bc5-58ae-46ca-90d5-717e008b6119.png">
Also can not navigate me to these elements that are marked in red. Which means you can not check if they say the right thing about the element

**Comment by a20caros** [comment]
<img width="477" alt="image" src="https://user-images.githubusercontent.com/81676918/166435202-112fda5c-d0d6-4818-b330-99c300a49b6e.png">
None of the buttons on the contribution page or the dropdown menu indicate that there are buttons or a dropdown menu as on the remaining page which is a problem

**Comment by a20caros** [comment]
<img width="541" alt="image" src="https://user-images.githubusercontent.com/81676918/166435741-304866ee-5385-4045-ba4e-682de060fa87.png">

Similarly, the screen reading tool does not indicate that it is a clickable button
